synapse/models/inet.py

```diff
@@ -2208,6 +2225,168 @@ def getModelDefs(self):
                         }),
                     )),
 
+                    ('inet:whois:ipquery', {}, (
+                        ('time', ('time', {}), {
+                            'ro': True,
```

Comment:
Secondary props which aren't derived from the primary property probably shouldn't be marked read-only

Reply:
updated